mono/mini/patch-info.h

```diff
@@ -70,3 +70,16 @@ PATCH_INFO(PROFILER_CLAUSE_COUNT, "profiler_clause_count")
  * function.
  */
 PATCH_INFO(METHOD_FTNDESC, "method_ftndesc")
+
+
+// Formerly MONO_PATCH_INFO_JIT_ICALL_ADDR + sprintf ("trampoline_func_%d", tramp_type)
```

Comment:
Put these into commit messages instead, it doesn't matter what they were previously.